MAUI/Busy-Indicator/Set-Header.md

```diff
@@ -199,3 +199,26 @@ The following screenshot illustrates the result of the above code.
 
 ![Font Customization](Images/SetHeader/Fontattributes.png)
 
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```

Comment:
The `FontAutoScalingEnabled` property is used to automatically scale the .NET MAUI Busy Indicator's title font size based on the operating system's text size. The default value of the `FontAutoScalingEnabled` property is `false`.

Reply:
Yes, changed.